Makefile

```diff
@@ -172,7 +174,13 @@ CLEANFILES += odh-manifests/*
 api-docs: crd-ref-docs ## Creates API docs using https://github.com/elastic/crd-ref-docs
 	$(CRD_REF_DOCS) --source-path ./ --output-path ./docs/api-overview.md --renderer markdown --config ./crd-ref-docs.config.yaml && \
 	egrep -v '\.io/[^v][^1].*)$$' ./docs/api-overview.md > temp.md && mv ./temp.md ./docs/api-overview.md
-	
+
+.PHONY: go-version-update
+golang-version-update:
+	for f in Dockerfiles/*; do \
+		sed  -i -e 's/\(ARG GOLANG_VERSION=\).*/\1$(GOLANG_VERSION)/g' $$f ; \
+	done
+
```

Comment:
but "make build" is still using the local Go, which can be different version from go.mod, right?

Reply:
Sure:
1) update the version in go.mod
2) run make golang-version-update
3) commit the changes to git and create a pull request